backend/api/Areas/Contacts/Mapping/Contact/OrganizationMap.cs

```diff
@@ -12,12 +10,14 @@ public void Register(TypeAdapterConfig config)
         {
             config.NewConfig<Entity.PimsOrganization, Model.OrganizationModel>()
                 .Map(dest => dest.Id, src => src.Id)
-                .Map(dest => dest.Name, src => src.OrganizationName)
-                .Map(dest => dest.Address, src => src.GetSingleAddress())
-                .Map(dest => dest.ContactName, src => src.GetFullName())
-                .Map(dest => dest.Landline, src => src.GetLandlinePhoneNumber())
-                .Map(dest => dest.Mobile, src => src.GetMobilePhoneNumber())
-                .Map(dest => dest.Email, src => src.GetEmail());
+                .Map(dest => dest.IsDisabled, src => src.IsDisabled)
```

Comment:
Can you confirm that this doesn't break the view tenant page?

Reply:
Yes, it will not break the tenant page. This model is exclusively used by the contacts page. Initially, it was used by the person section only so it was very minimal.